extensions/python/PythonScriptEngine.h

```diff
@@ -84,7 +48,7 @@ class PythonScriptEngine {
   PythonScriptEngine& operator=(const PythonScriptEngine& other) = delete;
   PythonScriptEngine& operator=(PythonScriptEngine&& other) = delete;
 
-  static void initialize() {}
+  static void initialize(const std::shared_ptr<Configure> &configuration, const std::shared_ptr<core::logging::Logger>& logger);
```

Comment:
Where is the implementation of this function?

Reply:
Good catch, it was removed, but the declaration remained, removed in 99359a0eda994fd5efa3095ebb76811ff1136f05